grammar.js

```diff
@@ -1146,6 +1148,25 @@ module.exports = grammar(C, {
       $.literal_suffix
     ),
 
+    alternative_unary: $ => prec.left(PREC.UNARY, seq(
+      field('operator', 'not'),
+      field('argument', $._expression)
+    )),
+
+    alternative_binary: $ => {
+      const table = [
+        ['or',  PREC.LOGICAL_OR],
```

Comment:
Please consider adding all of them. This should not make the PR any more complex and we will have full support for alternative operators.